src/main/cljc/jarl/builtins/registry.cljc

```diff
@@ -179,6 +179,9 @@
       ; semver
       "semver.compare"                    semver/builtin-semver-compare
       "semver.is_valid"                   semver/builtin-semver-is-valid
+      ; rego
+      "rego.metadata.chain"               (constantly true)
+      "rego.metadata.rule"                (constantly true)
```

Comment:
These will never be called, unless we're fed a broken/unexpected IR; but it doesn't hurt to have them here. Question is then if we should be less lenient and throw an error 🤔 . Alternatively the functions could return an empty list and object, respectively ..

Reply:
They will never be called, but I thought the compliance tests for these would be ignored unless we had them here. Looks like you're right though — the functions are not declared as dependencies to those plans, so this isn't really needed. I'll add a note to the README, and perhaps throw here as you suggested.